components/EmotionForm.js

```diff
@@ -0,0 +1,157 @@
+import Link from "next/link";
+import styled from "styled-components";
+
+const StyledH1 = styled.h1`
+  text-align: center;
+`;
+
+const StyledForm = styled.form`
+  border-radius: 10px;
+  margin: 1rem;
+  padding: 1rem;
+  background-color: ${({ $color }) => $color};
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  gap: 1rem;
+`;
+const StyledWrapper = styled.div`
+  display: flex;
+  width: inherit;
+  justify-content: space-between;
+`;
+
+const StyledSpan = styled.span`
+  font-size: 0.8rem;
+`;
+
+const StyledLink = styled(Link)`
+  text-decoration: none;
+  color: black;
+  margin: 1rem;
+  padding: 1rem;
+  border-radius: 8.5px;
+  &:hover {
+    background-color: ${({ $color }) => $color};
+  }
+`;
+const StyledSelect = styled.select`
+  border-radius: 6px;
+  padding: 0.3rem 0;
+`;
+
+const StyledTextarea = styled.textarea`
+  border: 1px solid black;
+  border-radius: 6px;
+`;
+
+const StyledSubmitButton = styled.button`
+  background-color: white;
+  border: 1px solid black;
+  border-radius: 6px;
+  padding: 0.5rem;
+  &:hover {
+    background-color: darkslateblue;
+    color: ${({ $color }) => $color};
+  }
+`;
+
+const StyledTextInput = styled.input`
+  padding: 0.3rem 0;
+  border-radius: 6px;
+  border: 1px solid black;
+`;
+
+export default function EmotionForm({
+  name,
+  color,
+  onAddEmotionDetails,
+  id,
+  slug,
+  emotionEntries,
+  subemotions,
+}) {
+  const correspondingEntry = emotionEntries.find((entry) => entry.id === id);
+
+  const correspondingId = id;
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    const data = Object.fromEntries(formData);
+
+    onAddEmotionDetails(data, correspondingId);
+  }
+
+  return (
+    <>
+      <StyledH1>Record your {name}</StyledH1>
+      <StyledForm $color={color} onSubmit={handleSubmit}>
+        <p>Date: {correspondingEntry.date}</p>
+        <p>Tension-Level: {correspondingEntry.tensionLevel}%</p>
+        <label htmlFor="emotion"></label>
```

Comment:
at least add an aria-label here to give some extra information.

Reply:
Hi, thanks for your feedback. Added some aria-labels. We will also do a whole user story for accessibility soon (hopefully in this sprint).